src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 // write your code here
+
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const sourceCell = row.cells[1];
+  const newCell = row.insertCell(row.cells.length - 1);
```

Comment:
The `insertCell` method is used with `row.cells.length - 1`, which inserts the new cell before the last cell. If the intention is to add the new cell at the end of the row, consider using `row.cells.length` as the index.